packages/wonder-blocks-grid/docs.md

```diff
@@ -99,12 +99,42 @@ Additionally, there are the following available components which will likely be
 
 Currently Grids are available at the following sizes (with their columns, gutter size, and margin sizes changing based on the size):
 
-| Size          | Columns | Gutter | Margin | Breakpoint                                                                                                                       |
-| ------------- | ------- | ------ | ------ | -------------------------------------------------------------------------------------------------------------------------------- |
-| small         | 4       | 16px   | 16px   | `max-width: 767px`                                                                                                               |
-| medium        | 8       | 32px   | 24px   | `min-width: 768px and max-width: 1023px`                                                                                         |
-| large         | 12      | 32px   | 24px   | `min-width: 1024px` (maximum content width: `20px`, after which the margins will continue to expand and content remain centered) |
-| internalTools | 12      | 32px   | 16px   | No breakpoint, must be explicitly specified in the `size` prop. Also no maximum content width.                                   |
+**Default Sizes (`GRID_DEFAULT_SIZES`)**
+
+| Size   | Columns | Gutter | Margin | Breakpoint                                                                                                                         |
+| ------ | ------- | ------ | ------ | ---------------------------------------------------------------------------------------------------------------------------------- |
+| small  | 4       | 16px   | 16px   | `max-width: 767px`                                                                                                                 |
+| medium | 8       | 32px   | 24px   | `min-width: 768px and max-width: 1023px`                                                                                           |
+| large  | 12      | 32px   | 24px   | `min-width: 1024px` (maximum content width: `1120px`, after which the margins will continue to expand and content remain centered) |
+
+Additionally, the following grid sizes are also available:
+
+**Internal Tools (`GRID_INTERNAL_SIZES`)**
+
+| Size  | Columns | Gutter | Margin | Breakpoint                                   |
+| ----- | ------- | ------ | ------ | -------------------------------------------- |
+| large | 12      | 32px   | 16px   | `min-width: 1px` (No maximum content width.) |
+
+**12-Column Modal (`GRID_MODAL_12_SIZES`)**
+
+| Size  | Columns | Gutter | Margin | Breakpoint                                     |
+| ----- | ------- | ------ | ------ | ---------------------------------------------- |
+| small | 4       | 16px   | 16px   | `max-width: 767px`                             |
+| large | 12      | 32px   | 64px   | `min-width: 768px` (No maximum content width.) |
```

Comment:
Is this correct?  It feels like either the grid should have a max width or the modal itself should have a max width.

Reply:
The only case where I could see us not wanting a max width on modals is the video modal, but we also want to navigate between video modals and the other content modals so all modals should have a max width.  @tyong is that correct?

Reply:
I think that it should be a modal that has a max width - until I hear otherwise I'm not going to put a max width on it and just let it grow to fill the container (which, in this case, is the modal).

Reply:
I've asked the question on slack.  It's easy enough to change once we know the answer.